src/frontend/test_runner/tests/testdata/join.yaml

```diff
@@ -8,39 +8,42 @@
   logical_plan: |
     LogicalProject { exprs: [$1, $2, $4, $5, $7, $8] }
       LogicalFilter { predicate: ($1 = $4) AND ($1 = $7) }
-        LogicalJoin { type: Inner, on: true }
-          LogicalJoin { type: Inner, on: true }
+        LogicalJoin { type: Inner, on: true, output_indices: [0, 1, 2, 3, 4, 5, 6, 7, 8] }
```

Comment:
I think this should only be shown when it isn't `0, 1, 2, ...`

Reply:
makes sense!